feature-pack-api/src/main/resources/schema/pm-provisioning-1_0.xsd

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<xs:schema xmlns="urn:wildfly:pm-provisioning:1.0" xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  attributeFormDefault="unqualified" elementFormDefault="qualified" targetNamespace="urn:wildfly:pm-provisioning:1.0"
+  version="2.0">
+
+  <xs:element name="installation" type="installation" />
+
+  <xs:complexType name="installation">
+    <xs:annotation>
+      <xs:documentation>
+        TODO: what is installation
+      </xs:documentation>
+    </xs:annotation>
+    <xs:choice maxOccurs="unbounded" minOccurs="1">
+      <xs:element name="universe" type="universe" />
+    </xs:choice>
+  </xs:complexType>
+
+  <xs:complexType name="universe">
```

Comment:
A minor note on the naming convention, I think in other schemas we follow the rule to name type as <element-name>Type. Would be good to continue it here. What do you think?